GliaWidgets/Public/Glia/Glia.Deprecated.swift

```diff
@@ -185,6 +185,45 @@ extension Glia {
         )
     }
 
+    /// Deprectated, use ``Glia.startEngagement(engagementKind:in queueIds:theme:features:sceneProvider:)`` instead.
```

Comment:
I would also mention that integrators should use `configure` method to pass `RemoteConfiguration`
Does it make sense?

Reply:
fixed [220f4f1](https://github.com/salemove/ios-sdk-widgets/pull/835/commits/220f4f17c6f363e055499306def4472320296c4b)